images/prow-test-image/Dockerfile-1.9

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
This is jenkins/test-images/Dockerfile-1.9 but with docker-in-docker support and a bootstrap runner that runs `service docker start` before bootstrap.